pkg/controller/volume/attachdetach/attach_detach_controller_test.go

```diff
@@ -155,7 +155,7 @@ func attachDetachRecoveryTestCase(t *testing.T, extraPods1 []*v1.Pod, extraPods2
 
 	stopCh := make(chan struct{})
 
-	pods, err := fakeKubeClient.Core().Pods(v1.NamespaceAll).List(metav1.ListOptions{})
+	pods, err := fakeKubeClient.CoreV1().Pods(v1.NamespaceAll).List(metav1.ListOptions{})
```

Comment:
Emm, are you expect performance improve from this change? (I am not quite sure if this fix make sense actually)